src/OrbitGl/App.cpp

```diff
@@ -999,65 +999,73 @@ void OrbitApp::SendErrorToUi(const std::string& title, const std::string& text)
   });
 }
 
-void OrbitApp::LoadModuleOnRemote(ModuleData* module_data,
-                                  std::vector<uint64_t> function_hashes_to_hook,
-                                  std::vector<uint64_t> frame_track_function_hashes,
-                                  std::string error_message_from_local) {
+orbit_base::Future<ErrorMessageOr<std::filesystem::path>> OrbitApp::LoadModuleOnRemote(
+    const std::string& module_file_path) {
   ScopedStatus scoped_status = CreateScopedStatus(absl::StrFormat(
-      "Searching for symbols on remote instance (module \"%s\")...", module_data->file_path()));
-  thread_pool_->Schedule(
-      [this, module_data, function_hashes_to_hook = std::move(function_hashes_to_hook),
-       frame_track_function_hashes = std::move(frame_track_function_hashes),
-       scoped_status = std::move(scoped_status),
-       error_message_from_local = std::move(error_message_from_local)]() mutable {
-        const auto result = GetProcessManager()->FindDebugInfoFile(module_data->file_path());
-
-        if (!result) {
-          SendErrorToUi(
-              "Error loading symbols",
-              absl::StrFormat("Did not find symbols locally or on remote for module \"%s\": %s\n%s",
-                              module_data->file_path(), error_message_from_local,
-                              result.error().message()));
-          main_thread_executor_->Schedule([this, module_data]() {
-            modules_currently_loading_.erase(module_data->file_path());
-          });
-          return;
-        }
+      "Searching for symbols on remote instance (module \"%s\")...", module_file_path));
```

Comment:
As written for a previous commit, I think this message should not use the parentheses.